CHANGELOG.md

```diff
@@ -1,4 +1,12 @@
+# dbt_netsuite_source v0.6.4
+
+
+## 🎉 Feature Updates 🎉
+- Brought in `accounting_book_id` (accountingbook) to the `stg_netsuite2__consolidated_exchange_rates` model. ([PR #36](https://github.com/fivetran/dbt_netsuite_source/pull/36))
+
 # dbt_netsuite_source v0.6.3
+
+## Under the Hood:
```

Comment:
I think this is an artifact of a merge conflict. This should probably be removed.

```suggestion
```